index.bs

```diff
@@ -671,63 +686,80 @@ When this method is invoked, the user agent MUST execute the following algorithm
+    and responses from the authenticators:
     <dl class="switch">
 
-    <dt>If the |adjustedTimeout| timer expires,</dt>
-    <dd>[=set/For each=] |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on |authenticator|
-        and [=set/remove=] |authenticator| from |issuedRequests|.</dd>
-
-    <dt>If any |authenticator| returns a status indicating that the user cancelled the operation,</dt>
-    <dd>
-        1. [=set/Remove=] |authenticator| from |issuedRequests|.
-        2. [=set/For each=] remaining |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on
-            |authenticator| and [=set/remove=] it from |issuedRequests|.
-    </dd>
-
-    <dt>If any |authenticator| returns an error status,</dt>
-    <dd>[=set/Remove=] |authenticator| from |issuedRequests|.</dd>
-
-    <dt>If any |authenticator| indicates success,</dt>
-    <dd>
-        1.  [=set/Remove=] |authenticator| from |issuedRequests|.
-        2.  Let |attestationObject| be a new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the
-            bytes of the value returned from the successful [=authenticatorMakeCredential=] operation (which is
-            <code>attObj</code>, as defined in [[#generating-an-attestation-object]]).
-        3.  Let |id| be |attestationObject|<code>.authData.attestation data.credential ID</code> (see [[#sec-attestation-data]]
-            and [[#sec-authenticator-data]]).
-        4.  Let |value| be a new {{PublicKeyCredential}} object associated with |global| whose fields are:
-            :   {{PublicKeyCredential/[[identifier]]}}
-            ::  |id|
-            :   {{PublicKeyCredential/response}}
-            ::  A new {{AuthenticatorAttestationResponse}} object associated with |global| whose fields are:
-                :   {{AuthenticatorResponse/clientDataJSON}}
-                ::  A new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the bytes of |clientDataJSON|.
-                :   {{AuthenticatorAttestationResponse/attestationObject}}
-                ::  |attestationObject|
-            :   {{PublicKeyCredential/clientExtensionResults}}
-            ::  A new {{AuthenticationExtensions}} object containing the [=extension identifier=] → [=client extension output=]
-                entries created by running each extension's [=client extension processing=] algorithm to create the [=client
-                extension outputs=], for each [=client extension=] in {{AuthenticatorResponse/clientDataJSON}}.clientExtensions.
-
-        5. [=set/For each=] remaining |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on
-            |authenticator| and [=set/remove=] it from |issuedRequests|.
-        6. Return |value| and terminate this algorithm.
-    </dd>
+        :   If the |adjustedTimeout| timer expires,
```

Comment:
I'm not manually-diffing this section, though. Please pull the editorial `<dt>`->`:` change into a separate PR so it's clear what's actually changed here.

Reply:
well, I have manually diffed it for you as a [rendered HTML diff here](http://kingsmountain.com/doc/diff/diff-webauthn-index-jeffh-fixup-algs-contd-3-a1b998f--from--master-3ee8ed5.html#createCredential), and I will do a text-only kdiff3 one tomorrow morning (i at least find the latter useful).   given above comment wrt while-we're-in-there markup improvements, I am loath (and do not have the time) to put all the <dd>...</dd> crud back in there.

Reply:
here's a [rendered side-by-side textual diff](http://kingsmountain.com/doc/diff/diff-webauthn-index-jeffh-fixup-algs-contd-3-a1b998f--from--master-3ee8ed5.text.pdf) (between this branch and the current state of the master branch) with which, in combination with the [rendered HTML daisydiff](http://kingsmountain.com/doc/diff/diff-webauthn-index-jeffh-fixup-algs-contd-3-a1b998f--from--master-3ee8ed5.html#createCredential), one can clearly see what has changed. daisydiff seems to get confused by things such as the "switch" stuff and so the textual diff helps to see what a reader will see.